frontend/lib/immediates/CMakeLists.txt

```diff
@@ -18,10 +18,14 @@
 target_sources(ChplFrontend-obj
                PRIVATE
 
-               ifa_vars.cpp
+               # C sources
+               complex-support.h
```

Comment:
Huh -- do header files usually get listed in sources? I don't recall that to be the case, but I could be forgetting.

Reply:
We have been listing them inconsistently. E.g. in frontend/lib/util/CMakeLists.txt. AFAIK it is harmless but unnecessary to list them.